src/main/scala/org/apache/mesos/chronos/scheduler/jobs/TaskUtils.scala

```diff
@@ -32,6 +32,15 @@ object TaskUtils {
     taskIdPattern.findFirstIn(taskIdString).nonEmpty
   }
 
```

Comment:
We might want to add a helper method to TaskUtils with the appropriate naming if we see this behavior repeatedly. Since this is only in 1 method, I think for readability sake, this makes more sense.